src/layouts/shortcodes/gsctl_version.html

```diff
@@ -1 +1 @@
-1.1.6
\ No newline at end of file
+null
```

Comment:
For some reason, these files keep getting created despite having removed this action in #2029